### DISCUSSION TITLE
Termux app default file location

### DISCUSSION THREAD

**Discussion by Anonymo-ambivert** [termux/termux-app#2862] [Q&A]
In termux app, **files/home** location available but **files/usr** location not available. I installed some packages like **apache2** and I get to know from Internet that -these packages are stored default in **files/usr/**  location.
Please someone provide suggestion to access **files/usr/** in my android termux app

**Comment by agnostic-apollo** [comment]
~~echo~~ `ls $PREFIX`

https://github.com/termux/termux-packages/wiki/Termux-file-system-layout

**Reply by Anonymo-ambivert** [reply]
echo command is just printing what we given as Input.But it's not actually locating the folder. If we give pwd i.e.,present loc , it should show files/usr .Its not happens in case of echo command
Please provide solution for this to access installed package loc

**Reply by agnostic-apollo** [reply]
echo was a left over typo. `$PREFIX` env variable defines the location  and expands to `/data/data/com.termux/files/usr` by default. Use it however you want in shell scripts.
